README.md

```diff
@@ -21,68 +20,70 @@ A minimalistic Flutter game engine.
-the code, and we are already quite happy with where we are (but there might still be further
-changes).
-
-Please use this version to get a preview of the new Flame version and also to give the team feedback
-about the new structure and/or features.
-
-The `main` branch is the bleeding edge of the v1 migration. The `master-v0.x` branch is the latest
-v0 release (where we are still merging some patches and crucial fixes).
-
-The current v1 release is
-<a title="Pub" href="https://pub.dev/packages/flame/versions#prerelease" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases" /></a>
-on pub. The latest stable version so far is
-<a title="Pub" href="https://pub.dev/packages/flame" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout" /></a>.
-Feel free to choose the one that better suits your needs.
+## Documentation
 
----
+The full documentation for Flame can be found on the 
+[docs.flame-engine.org](https://docs.flame-engine.org/).
 
-## Documentation
+To change the version of the documentation, use the version selector noted with `version:` in the
+top of the page.
 
 Note that the documentation in the main branch of this repo is newer than the latest released
 version.
 
-Here you can find the documentation for different versions:
-- Latest stable version: [Flame-engine website](https://flame-engine.org/)
-- Latest stable version: [GitHub docs](https://github.com/flame-engine/flame/tree/master-v0.x/doc)
-- Latest v1.0.0 version: [GitHub docs](https://github.com/flame-engine/flame/tree/1.0.0-releasecandidate.14/doc)
+Other useful links:
+ - [The official Flame site](https://flame-engine.org/).
+ - [Examples](https://examples.flame-engine.org/) of most features which can be tried out from your browser.
+ - [Tutorials](https://tutorials.flame-engine.org/) - A simple tutorial to get started.
+ - [API Reference](https://pub.dev/documentation/flame/latest/) - The generated dartdoc API reference.
 
-The complete documentation can be found [here](https://github.com/flame-engine/flame/tree/main/doc).
-
-We provide many examples of different features which can be tried out from your browser
-[here](https://flame-engine.github.io/flame/). You can also check the examples source code [here](https://github.com/flame-engine/flame/tree/main/examples).
-
-The official site for Flame, which also contains the documentation can be found
-[here](https://flame-engine.org/).
 
 ## Help
 
-We have a Flame help channel on Blue Fire's Discord, join it [here](https://discord.gg/5unKpdQD78).
-
-We also have a [FAQ](FAQ.md), so please search for your question there first.
-
-## Goals
-
-The goal of this project is to provide a complete set of out-of-the-way solutions for the common
-problems every game developed in Flutter will share.
+We have a Flame community and help channel on [Blue Fire's Discord](https://discord.gg/5unKpdQD78).
```

Comment:
```suggestion
There is a Flame community on the [Blue Fire's Discord server](https://discord.gg/5unKpdQD78) where you can ask any of your Flame related questions.
```

Reply:
You'd have to remove either `the` or the `'s` from Blue Fire